### ISSUE TITLE
treatment upload to BLR: progress

### ISSUE THREAD

**Issue by myrmoteras** [plazi/arcadia-project#136]
@gsautter when do we have the treatmentCitation as you have discussed and agreed with Terry implemented? 

Once this is ok, then we should go ahead to start uploading >100,000 treatments 
* starting from the most recent backwards.
* new taxon treatments
* add new treatments when they come in
* including GBIF ID for treatments

Can we already add DOIs for treatments that are referenced in the treatmentCitation and for which we have a treatment?

**Comment by myrmoteras** [comment]
@gsautter is the TreatmentCitation now fixed? If so, let's start the upload so we have next week a larger corpus?!

**Comment by gsautter** [comment]
Treatment citations have been fixed with yesterday morning's build, yes ... however, we still need to fix everything that was processed before that, I'm afraid, so I'm happy to upload the latest EJTs, but the other documents still need fixing.

**Comment by gsautter** [comment]
And we have 6 new treatments on Zenodo: http://tb.plazi.org/GgServer/srsStats/stats?outputFields=doc.uuid+doc.doi+doc.zenodoDepId+doc.name+doc.articleUuid+tax.name+tax.status&groupingFields=doc.uuid+doc.doi+doc.zenodoDepId+doc.name+doc.articleUuid+tax.name+tax.status&FP-doc.articleUuid=FFB680038C4BFFCD8374A277CE10947B&format=HTML

**Comment by myrmoteras** [comment]
super. 
does this mean, that all new treatments we start uploading now to TB, are also uploaded to BLR?

**Comment by gsautter** [comment]
Not automatically, as there is no way of telling an update to an older treatment from a new upload from now onward, at least not without considerable hacking ... I'll be a man in the middle for now and trigger the uploads from the console.

**Comment by myrmoteras** [comment]
I thought automatic upload if a new article is uploaded, such as an EJT or Zootaxa article, not individual treatments.

**Comment by gsautter** [comment]
Yes, but that doesn't change a thing ... update to existing and upload of new are still too similar, also at the article level ... I'll use the stats and he console for now.
